docs/tutorials/07_pegasos_qsvc.ipynb

```diff
@@ -26,7 +26,12 @@
    "cell_type": "code",
    "execution_count": 1,
    "id": "impressed-laser",
-   "metadata": {},
+   "metadata": {
```

Comment:
Can you just revert this file to the original form that was checked in since as noted its not really changed by this PR.

Reply:
Makes sense. I will do so.